[2021-08-04 13:28:06,038] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C 2021-08-04T13:28:01.221910+00:00 [queued]>
[2021-08-04 13:28:06,086] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C 2021-08-04T13:28:01.221910+00:00 [queued]>
[2021-08-04 13:28:06,087] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:28:06,087] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:28:06,087] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:28:06,122] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_C> on 2021-08-04T13:28:01.221910+00:00
[2021-08-04 13:28:06,135] {standard_task_runner.py:52} INFO - Started process 9209 to run task
[2021-08-04 13:28:06,166] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_C', '2021-08-04T13:28:01.221910+00:00', '--job-id', '1164', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmppwaozgxv', '--error-file', '/tmp/tmpalk3btwt']
[2021-08-04 13:28:06,166] {standard_task_runner.py:77} INFO - Job 1164: Subtask training_model_C
[2021-08-04 13:28:06,459] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_C 2021-08-04T13:28:01.221910+00:00 [running]> on host 04479ea62028
[2021-08-04 13:28:06,847] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:28:01.221910+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:28:01.221910+00:00
[2021-08-04 13:28:06,849] {python.py:151} INFO - Done. Returned value was: 6
[2021-08-04 13:28:07,025] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_C, execution_date=20210804T132801, start_date=20210804T132806, end_date=20210804T132807
[2021-08-04 13:28:07,270] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:28:07,294] {local_task_job.py:149} INFO - Task exited with return code 0
